scenes/client.gd

```diff
@@ -9,6 +9,14 @@ extends Node
 var _client = WebSocketClient.new()
 
 func _ready():
+	# HACK: Godot v4 beta14 Web build seems to run the djclient scene by default
+	# instead of the one specified in the Project Settings... so force a scene
+	# change!
+	if OS.get_name() == "Web":
+		print('forcing scene change to instrument selection')
+		get_tree().change_scene_to_file("res://scenes/instrument_selection/instrument_selection.tscn")
+		return
```

Comment:
We don't need this hack, it's just a project setting that needs to be changed. In Project settings/application/run, there's a field called "Main Scene.web".

Reply:
Ah, thanks. It was hidden under Advanced Settings :)